### ISSUE TITLE
Need scheduler to have Month slots with Year

### ISSUE THREAD

**Issue by manprit-sahota** [DHTMLX/scheduler#31]
I want to use dhtmlx scheduler but my requirement is that, I want Month slot to be display on header like January, February,etc and Year to be display on header. When I change year then again data from Month and year should be bind. 
![bprx2](https://user-images.githubusercontent.com/19249055/28972025-c6028c34-794b-11e7-935c-6a28095b670a.png) 

Also, I want 4 months to be display at a time. So I want to know that is it possible to achieve with dhtmlx scheduler? Also, my future requirement is to have 4 timeline in a month. So that task can be assign as per quarter in months. Review the attachment and let me know if it is possible or not? Please do needful asap.

**Comment by AlexKlimenkov** [comment]
Hi,
sure, the time step of the timeline is fully configurable and you can have variable time step: http://snippet.dhtmlx.com/15698bfd4
https://docs.dhtmlx.com/scheduler/timeline_view.html

>>Also, my future requirement is to have 4 timeline in a month. So that task can be assign as per quarter in months.
I'm not quite sure I understand, could you please clarify?

**Comment by manprit-sahota** [comment]
A great thanks to you @AlexKlimenkov for quick response. Its almost what I need to integrate. Just check out below image. I want that task can be assign in month quarters. My requirement is that: If I want to assign task T to an employee E then I can provide him time of 1 week, 2 week, 3 week or 4 week in a month (Work is assigned for 5 days, 10 days, 15 days or 20 days in month).
![image](https://user-images.githubusercontent.com/19249055/29038157-4685ca9a-7bc4-11e7-83cf-d2da6745421a.png)

**Comment by AlexKlimenkov** [comment]
Sorry for not returning to you earlier.
I can't see an easy way to implement it. The closest you can get is a second time-scale - https://docs.dhtmlx.com/scheduler/timeline_view.html#secondxaxis
 - you can have months at the top and weeks as the second scale (if needed, you can hide week scale with some styles, so for the user, there still will be single-line header). This, however, may not be exactly what you need. Such settings will align months scale cells to the start of week cells http://snippet.dhtmlx.com/6e2ee4d73 - i.e. February cell starts on first Monday here, which is actually the 6th February.

In short, I'm not sure that your setting can be done using public api. You either have to modify the source codes or you can contact our sales sales@dhtmlx.com for a custom development
